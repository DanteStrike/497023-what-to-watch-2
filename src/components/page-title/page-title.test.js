import React from "react";
import renderer from "react-test-renderer";
import PageTitle from "./page-title.jsx";


it(`Render correctly PageTitle component`, () => {
  const component = renderer
    .create(
        <PageTitle
          title="any"
        />
    ).toJSON();

  expect(component).toMatchSnapshot();
});
