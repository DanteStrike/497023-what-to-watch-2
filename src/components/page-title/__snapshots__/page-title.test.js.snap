// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly PageTitle component 1`] = `
<h1
  className="page-title user-page__title"
>
  any
</h1>
`;
